dask_kubernetes/operator/core.py

```diff
@@ -0,0 +1,167 @@
+import subprocess
+import json
+import re
+import time
+
+from distributed.core import rpc
+from distributed.deploy import Cluster
+
+from distributed.utils import Log, Logs
+
+from dask_kubernetes.auth import ClusterAuth
+from .daskcluster import (
+    build_cluster_spec,
+    build_worker_group_spec,
+)
+
+
+class KubeCluster2(Cluster):
+    """Launch a Dask Cluster on Kubernetes using the Operator"""
+
+    def __init__(
+        self,
+        name,
+        namespace="default",
+        image="daskdev/dask:latest",
+        replicas=3,
```

Comment:
I would be tempted to call this `n_workers` to be consistent with other cluster managers.